lib/util/mp4_parser.js

```diff
@@ -111,6 +111,12 @@ shaka.util.Mp4Parser = class {
   parseNext(absStart, reader, partialOkay, stopOnPartial) {
     const start = reader.getPosition();
 
+    // size(4 bytes) + type(4 bytes) = 8 bytes
+    if (start + 8 >= reader.getLength()) {
```

Comment:
Based on the parameter descriptions, I think this should depend on the stopOnPartial flag.  In some contexts, it should be an error to hit an incomplete box.  In LL-DASH context, I expect stopOnPartial should be true.  Is that correct?

Reply:
Yes, you are correct. stopOnPartial is true in LL-Dash context.